aws/resource_aws_backup_vault_notifications_test.go

```diff
@@ -0,0 +1,204 @@
+package aws
+
+import (
+	"fmt"
+	"log"
+	"testing"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/backup"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/terraform"
+)
+
+func init() {
+	resource.AddTestSweepers("aws_backup_vault_notifications", &resource.Sweeper{
+		Name: "aws_backup_vault_notifications",
+		F:    testSweepBackupVaultNotifications,
+	})
+}
+
+func testSweepBackupVaultNotifications(region string) error {
+	client, err := sharedClientForRegion(region)
+	if err != nil {
+		return fmt.Errorf("Error getting client: %w", err)
+	}
+	conn := client.(*AWSClient).backupconn
+
+	input := &backup.ListBackupVaultsInput{}
+
+	for {
+		output, err := conn.ListBackupVaults(input)
+		if err != nil {
+			if testSweepSkipSweepError(err) {
+				log.Printf("[WARN] Skipping Backup Vault Notifications sweep for %s: %s", region, err)
+				return nil
+			}
+			return fmt.Errorf("Error retrieving Backup Vault Notifications: %w", err)
+		}
+
+		if len(output.BackupVaultList) == 0 {
+			log.Print("[DEBUG] No Backup Vault Notifications to sweep")
+			return nil
+		}
+
+		for _, rule := range output.BackupVaultList {
+			name := aws.StringValue(rule.BackupVaultName)
+
+			log.Printf("[INFO] Deleting Backup Vault Notifications %s", name)
+			_, err := conn.DeleteBackupVaultNotifications(&backup.DeleteBackupVaultNotificationsInput{
+				BackupVaultName: aws.String(name),
+			})
+			if err != nil {
```

Comment:
Use `github.com/hashicorp/go-multierror`, collect errors and continue.
Look at `aws/resource_aws_sns_topic_test.go` for inspiration.